helm/mds/values.yaml

```diff
@@ -57,17 +57,17 @@ processors:
     triggeredBy:
       trigger: [event, telemetry]
   mds-trip-processor:
-    enabled: true
+    enabled: false
     version: latest
     migration: false
     triggeredBy:
-      cron: '0 * * * *'
+      cron: '* * * * *'
   mds-provider-processor:
-    enabled: true
+    enabled: false
     version: latest
     migration: false
     triggeredBy:
-      cron: '0 * * * *'
+      cron: '* * * * *'
```

Comment:
@jwtodd I believe we can completely remove the mds-[trip/event]-processor Deployments now.

Reply:
yeah but we still use this section of values.yaml to configure the k8s CronJob pods (see: helm/mds/processors-cronjob.yaml)

a measured amount of cleanup should be done here. @evanxadkins thoughts?

my $.02US ... have a simple templates/cronjob.yaml for all cronjob managed things

Reply:
@evanxadkins also, do we want to set the interval back to every hour?

Reply:
This should not be `* * * * *` (every minute) I think.